### ISSUE TITLE
Venue data update for BIZ2-0111

### ISSUE THREAD

**Issue by zwliew** [nusmodifications/nusmods#3604]
- Room Name: Biz2-0111
- Floor: 1
- Location: [1.3004393, 103.7709671](https://www.openstreetmap.org/?mlat=1.3004393&mlon=103.7709671#map=19/1.3004393/103.7709671)

**Venue does not exist in current version**

**Update proposed:**

```json
"BIZ2-0111": {
  "roomName": "Biz2-0111",
  "floor": 1,
  "location": {
    "x": 103.7709671,
    "y": 1.3004393
  }
}
```

**Comment by ravern** [comment]
Closed in #3652.

**Comment by ravern** [comment]
Closed in #3652.
